### ISSUE TITLE
mouse control

### ISSUE THREAD

**Issue by jmck** [phoboslab/jsmpeg-vnc#2]
I believe the server needs to run Administrator for it to allow mouse control (at least on Windows 8.1), documentation should likely mention this.

**Comment by phoboslab** [comment]
Yep, thanks for the notice! This seems to be a new requirement in Windows 8: http://stackoverflow.com/questions/22259936/c-move-mouse-in-windows-using-setcursorpos

Do you know if we can force the App to run with elevated rights? Or is this always something the user has to do himself (as in `right click -> run as administrator`)?

**Comment by tobbez** [comment]
You can do this by embedding a manifest. See https://msdn.microsoft.com/en-us/library/bb756929.aspx.

**Comment by jpmckearin** [comment]
Sorry to hijack this ancient issue but @jmck could you shoot me an email? josh@mckearin.codes. Thanks.
